config/application.rb

```diff
@@ -126,6 +128,8 @@ class Application < Rails::Application
     # The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
     # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
     # config.i18n.default_locale = :de
+    I18n.config.enforce_available_locales = false
+    I18n.config.available_locales = %w(de en en_GB es fr gl it ja ko pt_BR ru sv_SE zh_CN zh_TW)
```

Comment:
Since you're configuring I18n directly rather than going through Rails, perhaps move this straight into that initialiser so it's all together?